orders/urls.py

```diff
@@ -1,4 +1,7 @@
-from django.urls import path
+from django.urls  import path
+
+from orders.views import OrderView
 
 urlpatterns = [
+    path('/neworder', OrderView.as_view())
```

Comment:
```suggestion
    path('', OrderView.as_view())
```
```python
POST :8000/orders
```
이걸로도 충분히 파악이 가능할 것 같아요!